scripts/io_utils.py

```diff
@@ -14,3 +16,20 @@ def check_if_write(file_path, force, throw_warning=False):
             return False
     else:
         return True
+
+
+def read_csvs_with_chunksize(filename, chunksize=10000, **kwargs):
+    """
+    Read a CSV with an optionally passed chunksize to make reading large files easier.
+    Re-raises any exceptions (likely mostly going to be FileNotFound errors) so they
+    can continue to be handled how they currently are in the various locations.
+    """
+    try:
+        with pd.read_csv(filename, chunksize=chunksize, **kwargs) as reader:
+            dflist = []
+            for chunk in reader:
+                dflist.append(chunk)
+            df = pd.concat(dflist)
+            return df
```

Comment:
```suggestion
         return df
```

Should this be outside the `with` statement to make sure the filename connection is closed?